rr_bot/rhythm.py

```diff
@@ -142,7 +142,7 @@ class RegressionRhythm(Rhythm):
 
     logger_name = "RHYTHM:Regression"
 
-    def __init__(self, inertia, handstroke_gap=1):
+    def __init__(self, inertia, peal_speed=178, handstroke_gap=1):
```

Comment:
Possibly a comment that `peal_speed` is in minutes and `handstroke_gap` is fraction of inter bell gap?

Reply:
Done!